subsystem/src/subsystem.rs

```diff
@@ -27,37 +30,29 @@ pub trait Subsystem: 'static + Send + Sized {
 pub struct SubsystemConfig {
     /// Subsystem name
     pub subsystem_name: &'static str,
-    /// Capacity of the call request channel
-    pub call_queue_capacity: usize,
 }
 
 impl SubsystemConfig {
-    const DEFAULT_CALL_QUEUE_CAPACITY: usize = 64;
     const DEFAULT_SUBSYSTEM_NAME: &'static str = "<unnamed>";
 
     /// New configuration with given name, all other options are defaults.
     pub(crate) fn named(subsystem_name: &'static str) -> Self {
-        Self {
-            subsystem_name,
-            call_queue_capacity: Self::DEFAULT_CALL_QUEUE_CAPACITY,
-        }
+        Self { subsystem_name }
     }
 }
 
 impl Default for SubsystemConfig {
     fn default() -> Self {
-        Self {
-            subsystem_name: Self::DEFAULT_SUBSYSTEM_NAME,
-            call_queue_capacity: Self::DEFAULT_CALL_QUEUE_CAPACITY,
-        }
+        let subsystem_name = Self::DEFAULT_SUBSYSTEM_NAME;
```

Comment:
It isn't related to your pull request, but I'm curious: why do we even have the `DEFAULT_SUBSYSTEM_NAME` constant? As far as I can see, we always use explicitly named subsystems even in the tests.

Reply:
Good point. Will have a look and possibly remove it if it turns out it's indeed not used.

Reply:
Removed